models/food.js

```diff
@@ -1,11 +1,24 @@
+const MealFood = require('../models').MealFood
+
 'use strict';
 module.exports = (sequelize, DataTypes) => {
   const Food = sequelize.define('Food', {
     name: DataTypes.STRING,
     calories: DataTypes.INTEGER
   }, {});
-  Food.associate = function(models) {
-    Food.hasMany(models.MealFoods);
+  Food.associate = function(m) {
+    Food.belongsToMany(m.User, {through: m.MealFood, foreignKey: m.FoodId});
+    Food.belongsToMany(m.Meal, {through: m.MealFood, foreignKey: m.FoodId});
   };
+  // Food.mealFoods = function(mealId, userId) {
```

Comment:
Did you end up finding any way to call on associated models?

Reply:
No, I couldn't get it working after a couple hours, so I just searched the associations manually.